templates/Web/_composition/Vue/Page.Vue.AddConstantsList/src/constants$wts.Page.Vue.List_gpostaction.js

```diff
@@ -6,7 +6,7 @@ CONSTANTS.ERROR_MESSAGE = {};
 CONSTANTS.ERROR_MESSAGE.LIST_DELETE = "Request to delete list item failed:";
 CONSTANTS.ERROR_MESSAGE.LIST_ADD = "Request to add list item failed:";
 CONSTANTS.ERROR_MESSAGE.LIST_GET = "Request to get list items failed:";
-CONSTANTS.ERROR_MESSAGE.LIST_EMPTY_MESSAGE = "Please enter a valid message";
+CONSTANTS.ERROR_MESSAGE.LIST_EMPTY_MESSAGE = "Input must be at least 1 character long.";
```

Comment:
Should we change this for React and Angular as well?

Reply:
Angular hard coded the value, so I'm pretty sure it wont use this constant. And yes we probably should do this for React, but in a different PR.